queries/leo/highlights.scm

```diff
@@ -0,0 +1,180 @@
+(variable) @variable
+
+[
+ "assert"
+ "assert_eq"
+ "assert_neq"
+ "block"
+ "console"
+ "const"
+ "constant"
+ "finalize"
+ "in"
+ "let"
+ "mapping"
+ "private"
+ "program"
+ "public"
+ "record"
+ "self"
+ "struct"
+] @keyword
+
+[
+	"constant"
+	"public"
+	"private"
+] @type.qualifier
+
+"self" @variable.builtin
+
+[
+ "transition"
+ "function"
+ "inline"
```

Comment:
inline could be `@storageclass` but correct me if it is a keyword to define functions

Reply:
According to the docs:
> An inline function is declared as `inline {name}() {}`